README.md

```diff
@@ -1,21 +1,82 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
```

Comment:
Please write it correctly and properly. 
* not only for ledger
* not only for PITR
* What is `PAUSE` and `UNPAUSE`?

```suggestion
It will help you to pause Scalar DL Ledger cluster or Scalar DB server cluster to create a transactionally-consistent backup.
```

Reply:
Thank you for your suggestion.
Document updated with the above image.